Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/BaseLinearElasticityFEMForceField.inl

```diff
@@ -48,12 +92,18 @@ void BaseLinearElasticityFEMForceField<DataTypes>::init()
         msg_info() << "link to Topology container should be set to ensure right behavior. First Topology found in current context will be used.";
         l_topology.set(this->getContext()->getMeshTopologyLink());
     }
+
+    checkYoungModulus();
+    checkPoissonRatio();
 }
 
 template <class DataTypes>
 void BaseLinearElasticityFEMForceField<DataTypes>::setPoissonRatio(Real val)
 {
-    this->d_poissonRatio.setValue(val);
+    VecReal newPoissonRationList;
+    newPoissonRationList.resize(1);
+    newPoissonRationList[0] = val;
+    d_poissonRatio.setValue(newPoissonRationList);
```

Comment:
I'm surprised, I would have expected:
d_poissonRatio.setValue({val});

Reply:
Again, taken from the original code. I'll change it. Thanks